cxx4opencl/diff2openclc.txt

```diff
@@ -4,17 +4,21 @@
 
 === Difference to OpenCL C
 
-{cpp} for OpenCL aims to achieve backwards compatibility with OpenCL C
-for the majority of features. However, it is a different
-language that is derived from {cpp} and therefore it inherits
-its fundamental design principles. Hence {cpp} for OpenCL
-deviates from OpenCL C in the same areas where {cpp} deviates
-from C.
+{cpp} for OpenCL provides backwards compatibility with OpenCL C
+for the majority of features. However, there are a number of
+exceptions that are described in this section. Some of them come
+from the nature of C++ but others are due to improved behaviour
+of a kernel language.
 
-This section describes the main differences between {cpp} for
-OpenCL and OpenCL C.
+==== C++ related differences
 
-==== Implicit Conversions
+{cpp} for OpenCL is a different language derived from {cpp} and
+therefore it inherits its fundamental design principles. Hence
```

Comment:
its is ambiguous. I think it would be better to say c++s fundamental design principles